src/Fixtures/index.js

```diff
@@ -1434,17 +1328,35 @@ export default {
         'https://cdn3.sportngin.com/attachments/photo/be7e-106813275/Screen_Shot_2018-07-19_at_3.53.39_PM_large.png',
       description:
         "This training session aims at practicing your mark skills. Don't be broken to easily, generate dangerous passes",
-      drills: [1, 1004, 1005, 2003],
+      drills: [5, 1004, 1005, 2003],
     },
+  ],
+  blocs: [
     {
-      id: 5,
-      illustrationType: 'Martin',
-      title: 'Coed 2 Long v2',
```

Comment:
I had added this one to have 2 trainings on the same topic, with shared drills, which could be part of the same program/bloc

Maybe you should bring it back since you also use it in a bloc